bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "url",
+  "main": "url.js",
+  "homepage": "https://github.com/Polymer/URL",
+  "authors": [
+    "https://github.com/Polymer/URL/graphs/contributors"
+  ],
+  "description": "URL parser in JavaScript",
+  "keywords": [
+    "url",
+    "urlsearchparams",
+    "whatwg",
+    "w3c"
+  ],
+  "license": "https://github.com/Polymer/URL#license",
```

Comment:
@arv https://github.com/Polymer/URL#license links to the 
```md
# License
This is public domain.
```
at `README.md`
not to the Polymer's license